pkg/virt-operator/install-strategy/create.go

```diff
@@ -731,6 +731,61 @@ func createOrUpdateConfigMaps(kv *v1.KubeVirt,
 
 	return nil
 }
+func rolloutNonCompatibleCRDChanges(kv *v1.KubeVirt,
```

Comment:
can we get a unit test in kubevirt_test that exercises this behavior? or maybe it's already exercised somewhere?

Reply:
added some to create_test.go